webhooks-extension/src/components/WebhookDisplayTable/WebhookDisplayTable.js

```diff
@@ -191,28 +342,36 @@ export class WebhookDisplayTable extends Component {
               )}
               />
           </div>
+
+          <div className="modal">
+            <Modal open={this.state.showDeleteDialog}
+              className=".delete-modal"
+              modalLabel="Are you sure?"
+              modalHeading="Really delete webhook(s)?"
+              primaryButtonText="Delete webhook"
+              secondaryButtonText="Cancel delete"
+              danger={true}
+              onRequestSubmit={this.handleDeleteWebhook}
+              onSecondarySubmit={this.showDeleteDialogHandlerInvisible}
+              onRequestClose={this.showDeleteDialogHandlerInvisible}
+              >
+            <label>
+              <Checkbox
+                data-modal-primary-focus
+                checked={this.state.checked}
+                onChange={this.togglePipelineRunClicked}
+              />
+              <span>Delete PipelineRuns?</span>
```

Comment:
I would change this to "Delete PipelineRuns associated with selected webhooks" - Or possibly, a sentence above the tick box stating ... "Selecting the checkbox below will cause the deletion of associated PipelineRuns" and then leave the checkbox label as Delete PipelineRuns